gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Apr 24 23:57:46 CEST 2019
+#Fri Sep 15 11:38:13 CEST 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```

Comment:
This should match the version listed in `build.gradle`/buildtools, so 8.1.1, or you should just go for the latest version, if it works.

Reply:
Done